api/responses.py

```diff
@@ -12,13 +12,13 @@ def magic_link_201_response(magic_link_dict: dict):
     return (
         make_response(
             {
-                "accountId": magic_link_dict.get("accountId"),
-                "iat": magic_link_dict.get("iat"),
-                "exp": magic_link_dict.get("exp"),
-                "token": magic_link_dict.get("token"),
-                "link": magic_link_dict.get("link"),
-                "redirectUrl": magic_link_dict.get("redirectUrl"),
-                "key": magic_link_dict.get("key"),
+                "accountId": magic_link_dict["accountId"],
```

Comment:
⛏️ You can do: 
`new_dict = {**old_dict}`

Reply:
https://medium.com/swlh/how-to-pack-and-unpack-data-in-python-tuples-and-dictionaries-55d218c65674

Reply:
Retained for visibility on response contents